src/github.com/getlantern/flashlight/proxiedsites/proxiedsites.go

```diff
@@ -52,6 +56,24 @@ func Configure(cfg *proxiedsites.Config, proxyAddr string) {
 	startMutex.Unlock()
 }
 
+func updateDetour(delta *proxiedsites.Delta) {
+	// TODO: subscribe changes of geolookup and set country accordingly
```

Comment:
Significant refactor is required to eliminate globals and unnecessary dependencies. I'll log a new issue for it.